harvard-university-of-leeds.csl

```diff
@@ -32,7 +36,7 @@
   </macro>
   <macro name="author">
     <names variable="author">
-      <name and="text" delimiter-precedes-last="never" initialize-with="." name-as-sort-order="all"/>
+      <name and="text" delimiter-precedes-last="never" et-al-min="0" initialize-with="." name-as-sort-order="all"/>
```

Comment:
Hi, I was sent the updates by someone else - I'll check what is intended in the bibliography and let you know.
What's the behaviour when ```et-al-min``` isn't defined?